geonode_mapstore_client/templates/geonode-mapstore-client/app/geostory.html

```diff
@@ -1,4 +1,5 @@
 {% load static %}
+{% load client_version %}
 
 {% include '../_geonode_config.html' with is_embed=is_embed is_app=True plugins_config_key=plugins_config_key|default:'geostory' is_new_resource=is_new|default:'true' %}
-<script id="ms2-api" src="{% static 'mapstore/dist/gn-geostory.js' %}"></script>
+<script id="ms2-api" src="/static/mapstore/dist/gn-geostory.js?{% client_version %}"></script>
```

Comment:
@marthamareal let's please use everywhere the notation

`{% static 'mapstore/dist/gn-XXX.js' %}`

Reply:
in case you can simply use {% static 'mapstore/dist/gn-geostory.js' %}?{% client_version %}

Reply:
updated this